Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -171,6 +174,12 @@ end Neg
 
 section Const
 
+/-- A class which encodes that constant functions are members of `Lp`.
+This has instances when `p := ∞` or `μ.IsFiniteMeasure`. -/
+class MemLp.Const {α : Type u_1} {m0 : MeasurableSpace α} (p : ℝ≥0∞)
+  (μ : Measure α) where
+  eLpNorm_const_lt_top' (c : ENNReal) (hc : ‖c‖ₑ ≠ ∞) : eLpNorm (fun _ ↦ c) p μ < ∞
```

Comment:
```suggestion
class MemLp.Const {α : Type u_1} {m0 : MeasurableSpace α} (p : ℝ≥0∞)
    (μ : Measure α) where
  eLpNorm_const_lt_top' (c : ℝ≥0∞) (hc : ‖c‖ₑ ≠ ∞) : eLpNorm (fun _ ↦ c) p μ < ∞
```
Also, can you please add theorems of the following forms:
```
MemLp.const p μ ↔ p = 0 ∨ p = ∞ ∨ μ.IsFiniteMeasure

p ≠ 0 → (MemLp.const p μ ↔ p = ∞ ∨ μ.IsFiniteMeasure)
```
as well as as alias for the first direction of the other one (although you should probably write it by hand so that `MemLp.const p μ` can be taken as a type class argument instead of an explicit argument).

Reply:
Ah. This is a good idea.

Reply:
What do you mean by an alias for the first direction of the other one?